providers/github/resources/github_org.go

```diff
@@ -161,7 +157,7 @@ func (g *mqlGithubOrganization) owners() ([]interface{}, error) {
 			id = *member.ID
 		}
 
-		r, err := CreateResource(g.MqlRuntime, "github.user", map[string]*llx.RawData{
+		r, err := g.MqlRuntime.CreateResource(g.MqlRuntime, "github.user", map[string]*llx.RawData{
```

Comment:
This looks strange to me.

I'm only aware of `CreateResource` usage, not `g.MqlRuntime.CreateResource`. The runtime is now passed in as a parameter.